src/Parameters/CreateMeetingParameters.php

```diff
@@ -869,6 +899,68 @@ public function setFreeJoin($freeJoin)
         return $this;
     }
 
+    /**
+     * @return bool
+     */
+    public function isBlockedJoin()
```

Comment:
Can we change all method names to reflect the actual api? So instead of `isBlockedJoin`, use `isGuestPolicyAlwaysDeny`. In my opinion it's easier to understand and the length should be no problem with autocomplete.

Reply:
Yes, feel free to change it. We implemented these methods names, but i'm sure that there are better names :)

Reply:
@sualko isGuestPolicyAlwaysDeny method name should easily readable for developers so i agreed with you.

Reply:
I remember why we didn't use a similar name like your suggestion:

If BBB change values of guestPolicy new suggestion names should be upgrading with the new values.

Pj:

Now if BBB string value is  "ALWAYS DENY" and method will be isGuestPolicyAlwaysDeny()

On the future, if BBB changes the value to "BLOCKED" our method will be renamed by isGuestPolicyBlocked() and we will break the backwards compatibility.

I think that we should use method names that abstract us from the value defined by BBB but that always refer to the functionality to which it relates, hence we used "blocked" as a generic name.

¿ what do you think about this?

Reply:
1. Why should BBB change the value?
2. If that would be a problem you would need to make everything abstract, which is no usable way.

Reply:
They can change the code when they need it, either by a simple code refactoring (to unify the nomenclature of the values) or by a new improvement. This is beyond our control as it is a third party library.

In my opinion, I think that trying to abstract ourselves from those values ​​allows us to adapt to that possibility of very easy change (just change the constant) and for developers who use the PHP API it would be transparent, since the method would not be modified.

However, both options are valid.

Reply:
> They can change the code when they need it

But that's true for every piece of the api, so I don't understand why we should abstract here and not for example also change `createMeeting` to `createRoom`. If it's ok, I would change those values to reflect the official API documentation.

Reply:
Yes, it's ok ;)